float32 cpu_usage
float32 gpu_usage
float32 mem_usage
int64 mem_total
